File: pyround1/1051.高度检查器.py
#
# @lc app=leetcode.cn id=1051 lang=python3
#
# [1051] 高度检查器
#

# @lc code=start
class Solution:
    def heightChecker(self, heights: List[int]) -> int:
        res = sorted(heights)
        count = 0
        for i in range(len(heights)):
            if res[i] != heights[i]:
                count += 1
        return count
# @lc code=end

File: 1051.高度检查器.cpp
/*
 * @lc app=leetcode.cn id=1051 lang=cpp
 *
 * [1051] 高度检查器
 */

// @lc code=start
class Solution {
public:
    int heightChecker(vector<int>& heights) {
        vector<int> tmp = heights;
        sort(tmp.begin(), tmp.end());
        int count = 0;
        for(int i = 0; i < tmp.size(); i++){
            if(tmp[i] != heights[i])
                count++;
        }
        return count;
    }
};
// @lc code=end

